### PR TITLE
parameterize zos in identity

### PR DESCRIPTION
**Pull request by xmonader**
parameterize zos  in identity so if we want to switch identity at certain point we can

### CODE REVIEW

File: jumpscale/sals/zos/__init__.py [hunk 1]
```diff
@@ -1,4 +1,34 @@
...
+    Returns:
+        [type]: [description]
+    """
+    global INSTANCES, ME_INSTANCE
+    from jumpscale.loader import j
     from .zos import Zosv2
 
-    return Zosv2()
+    if not identity_name:
+        identity_name = j.core.identity.me.instance_name
```
Comment (waleedhammam) [inline jumpscale/sals/zos/__init__.py:20]: I think If I have no default identity this one gonna crash ?

Comment (xmonader) [reply jumpscale/sals/zos/__init__.py:24]: Addressed, also handled the case of using identity_name that doesn't exist

**Review by waleedhammam — APPROVED** [review]
When merging, we should make sure to merge back with development and make the new usage,  some new changes on were introduced after the pr was created

**Comment by xmonader** [comment]
I updated to the latest please let's give it some prio

**Review by abom — APPROVED** [review]

**Pull request merged**